docs/standards/VEP/vep-1155.md

```diff
@@ -0,0 +1,538 @@
+---
+title: "VEP-1155: Multi-Token"
+---
+
+```preamble
+VEP: 1155
+author: Evgeny Shatalov <evgeny.a.shatalov@gmail.com>, Aleksei Kolchanov <a.kolchanov@numi.net>
+status: Review
+type: Informational
```

Comment:
I think it should be Contract subtype of Standards type. I see, that description of Contract subtrack is not good for such kind of VEP, but I think we need to change it somehow to show, that contract track also includes erc-like standards. So you can change type to Contract